examples/vue.test.js

```diff
@@ -2600,6 +2600,40 @@ describe('Vue', function() {
       // acquit:ignore:end
     });
   });
+  it('vue-websocket', async function() {
+    // acquit:ignore:start
+    this.timeout(10000);
+    const browser = await puppeteer.launch({ headless: false });
+    const page = await browser.newPage();
+    const fn = function() {
+    // acquit:ignore:end
+      const app = new Vue({
+        data: () => ({ time: null }),
+        errorCaptured: function(err) {
+          console.log('Caught top-level error', err.message);
+          return false;
+        },
+        template: `
+          <div>
+            <h1>Hello</h1>
```

Comment:
I'd just get rid of this `<h1>`. Unnecessary code is bad because it makes it difficult to figure out what code is actually necessary